src/controllers/userController.js

```diff
@@ -13,29 +13,37 @@ const getAllUsers = async (req, res) => {
   // Find all users with the role 'user' in the database and exclude the 'password' field
   const users = await User.find({ role: 'user' }).select('-password');
   res.status(StatusCodes.OK).json({ users }); // Send a JSON response with the status code 200 OK and the users
+  /*
+     #swagger.summary = 'Fetch all registered users in a database whose role is user (exclude admins)'
```

Comment:
Looks great! We can also add a little blurb in the description (when they expand that endpoint in the docs) to indicate that this endpoint can only be called by admins. Swagger uses some flavour of [Markdown](https://www.markdownguide.org/basic-syntax/), so we can use `**` to bold text:

```suggestion
     #swagger.summary = 'Fetch all registered users in a database whose role is user (exclude admins)'
     #swagger.description = '**ROLE REQUIRED:** admin'
```

Reply:
@akosasante thank you!! so glad to learn new things!!!